code/entities/cooking/hotdog-roller/HotdogRoller.Hotdog.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Sandbox;
+
+namespace Cinema.HotdogRoller;
+
+public partial class HotdogCookable : ModelEntity
```

Comment:
Nit / Not Required:

Mildly confusing file name. `HotdogRoller.Hotdog.cs` contains the `HotdogCookable` entity